organization/src/it/scala/com/improving/app/organization/OrganizationServerSpec.scala

```diff
@@ -31,90 +31,6 @@ class OrganizationServerSpec extends ServiceTestContainerSpec(8082, "organizatio
     }
   }
 
-  it should "gracefully handle bad requests that fail at service level" taggedAs Retryable in {
-    withContainers { containers =>
-      val client = getClient(containers)
-
```

Comment:
The service level test is a weird one to have now. With on construction message validation there is no possible error state for the service because there was previously only one errors state, when entityId was not provided. On top of which, as discussed last week, if validation still fails we want to handle that in Gateway when translating from http to grpc (possibly as part of a complex flow of actions and/or replies to user).

Perhaps we can think out the box a bit here and come up with some other reason to get a service level failure... but the service doesn't do much of anything other than cluster sharding, result transformation, and exception handling for invalid arguments, the latter which are now impossible and the former of which are safe operations generally...

Reply:
I will add the invalid state attempt on entity level though

Reply:
Is there a way to force it to timeout via test?

Reply:
I guess just initiate it with an ActorSystem that errors?

Reply:
Whatever works. We can punt if it turns out to be hard, but it's good practice to have at least one test of other-than-perfect behavior, if only to check what would happen in that code path so we have some idea what to expect.